main-packages-list.ocp

```diff
@@ -47,7 +47,7 @@ python3-paste-deploy >= 2.0.1-4.el8ost
 python3-pint >= 0.10.1-1.el8ost
 python3-proliantutils >= 2.12.0-0.20211012162915.abc6432.el8
 python3-pyghmi >= 1.5.14-2.1.el8ost
-python3-scciclient >= 0.9.1-0.20210827164043.21a184b.el8
+python3-scciclient = 0.10.0-0.20211117144100.22064a9.el8
```

Comment:
Any particular reason to swap from ">=" to "="?

Reply:
this change has been done to avoid an unpleasant situation where an older version was installed from the OSP repo
that's why we're pinning a specific version now, ,and we're probably do the same for other ironic related packages too, at least until we complete the removal of OSP repo dependency

Reply:
Good to know =)